geoai/change_detection.py

```
@@ -111,39 +124,64 @@ def _get_change_confidence_threshold(self) -> float:
 
     def set_hyperparameters(
         self,
-        change_confidence_threshold: int = 155,
-        auto_threshold: bool = False,
-        use_normalized_feature: bool = True,
-        area_thresh: float = 0.8,
-        match_hist: bool = False,
-        object_sim_thresh: int = 60,
-        bitemporal_match: bool = True,
+        change_confidence_threshold: Optional[int] = None,
+        auto_threshold: Optional[bool] = None,
```

Ruff (0.15.20)

[warning] Boolean-typed positional argument in function definition

(FBT001)